edlib/src/edlib.cpp

```diff
@@ -1,4 +1,4 @@
-#define EDLIB_DLL_EXPORT
+#define EDLIB_BUILD
```

Comment:
Can this line be removed?

Reply:
In the first implementation of this PR, this definition was injected at build time. So yes it can be removed in .cpp, but meson.build should be updated accordingly.

Reply:
But doesn't it make sense to have it here? Isn't this the only situation where this symbol should be defined?